src/zc/buildout/easy_install.py

```diff
@@ -1521,7 +1521,11 @@ def _pyscript(path, dest, rsetup, initialization=''):
 
 _interactive = True
 if len(sys.argv) > 1:
-    _options, _args = __import__("getopt").getopt(sys.argv[1:], 'ic:m:')
+    # The Python interpreter wrapper allows some of the options that a
+    # "regular" Python interpreter accepts, but it only acts on the -i, -c
+    # and -m options. The other option(s) are ignored. See
+    # https://github.com/buildout/buildout/issues/627 for more information.
```

Comment:
@janwijbrand I used to refer to issues on websites, but they do eventually disappear. Would you mind explaining about VSCode need here ?